eth/api_backend.go

```diff
@@ -408,25 +408,19 @@ func (b *EthApiBackend) StateAtBlock(ctx context.Context, block *types.Block, re
 
 func (s *EthApiBackend) GetRewardByHash(hash common.Hash) map[string]map[string]map[string]*big.Int {
 	header := s.eth.blockchain.GetHeaderByHash(hash)
+	err := errors.New("header hash not found")
```

Comment:
This line is useless sometimes, such as when `return rewards`.

Reply:
relocate this error declaration and put a missing hash into the message, so that whenever it logs, we know which hash is misisng